docs/source/guides/libraries.rst

```diff
@@ -67,12 +67,256 @@ Python library
 Where to get
 ^^^^^^^^^^^^
 
+Prerequirements:
+
+- cmake(3.11 or higher)
+- git
+- g++
+- boost(1.65 or higher, only system and filesystem)
```

Comment:
`system` and `filesystem` are not required for shared_model, because only headers are used.